code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -38,14 +38,16 @@
 			return BULLET_ACT_FORCE_PIERCE
 
 /obj/projectile/plasma/adv
-	damage = 25 //SKYRAT EDIT CHANGE - ORIGINAL: 7
+	damage = 7
 	range = 5
 	mine_range = 5
+	dismemberment = 30 //SKYRAT EDIT CHANGE
 
 /obj/projectile/plasma/adv/mech
-	damage = 45 //SKYRAT EDIT CHANGE - ORIGINAL: 10 - Seriously? Do you have no respect for dead space?
+	damage = 10
 	range = 9
-	mine_range = 5 //SKYRAT EDIT CHANGE - ORIGINAL: 3
+	mine_range = 7 //SKYRAT EDIT CHANGE - ORIGINAL: 3
+	dismemberment = 40 //SKYRAT EDIT CHANGE
```

Comment:
That's incredibly high and makes this much more powerful in a PVP scenario since 80% of the time, after a mere two hits, your enemy will be missing a limb, which is a big amount of damage and severely hinders said opponent.

Reply:
:thinking: For real.

Reply:
Also changed by cutting everything by about 10%-or-whatever, yeah.